### DISCUSSION TITLE
Later list

### DISCUSSION THREAD

**Discussion by ayr-ton** [element-hq/element-meta#2122] [Ideas]
#### What would you like to do?
To have a later button that send a message to a later list. Similar to a TODO.

#### Why would you like to do it?
After acknowledging a message, normally I like to add to my to-do to reply later or create some task. Having this in element would be great so I don't need to keep control of lists from element in other apps. 

#### How would you like to achieve it?
- Long press a message (or select with secundary click for web/desktop)
- Select "later"
- Message is marked as read and sent to later list
- On the later list, see full list
- Mark message as complete when completed

#### Optional
- Reorder later list itens 
- Sync later list with all element devices (other feature)

### Have you considered any alternatives?

- Slack have this
- Hey e-mail client allow creating more complex lists and kanbans
- Other TODO list apps work but I can't link it with element 

### Additional context

This will make favorite lists more alive

### Are you willing to provide a PR?

No

**Comment by kittykat** [comment]
I've moved this issue to a discussion in element-meta as we need to reach a cross-platform decision on how this will be addressed. At the moment, there are plans to work on this in 2024 :+1:

**Reply by ayr-ton** [reply]
@kittykat This is awesome ❤️ 
Count on me for writing any specs that could help
